Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -708,12 +1006,13 @@ public class KotlinTokenizer: SwiftTokenizer {
         }
         
         // Other returns must be suffixed with call name
-        if let callExpression = expression.lexicalParent as? FunctionCallExpression,
-            let memberExpression = callExpression.postfixExpression as? ExplicitMemberExpression {
+        if let callExpression = expression.lexicalParent as? FunctionCallExpression { //,
+        //    let memberExpression = callExpression.postfixExpression as? ExplicitMemberExpression {
             while let returnIndex = tokens.index(where: { $0.value == "return" }) {
                 tokens.remove(at: returnIndex)
                 tokens.insert(expression.newToken(.keyword, "return@"), at: returnIndex)
-                tokens.insert(expression.newToken(.identifier, memberExpression.identifier), at: returnIndex + 1)
+//                tokens.insert(expression.newToken(.identifier, memberExpression.identifier), at: returnIndex + 1)
+                tokens.insert(expression.newToken(.identifier, callExpression.postfixExpression.description), at: returnIndex + 1)
```

Comment:
I do not understand the change. Could you provide and example of the code that was not translating properly before and after?